File: examples/availability/retrieve_ipfs/retrieve_ipfs.py
import os
from bloock import bloock
from bloock.client.record import RecordClient
from bloock.entity.availability.ipfs_loader import IpfsLoader

if __name__ == "__main__":
    bloock.api_key = os.environ["API_KEY"]

    record_client = RecordClient()

    pid = "publish uuid result"
    record = record_client.from_loader(IpfsLoader(pid)).build()

File: examples/availability/retrieve_ipfs/retrieve_ipfs.php
<?php

require "./vendor/autoload.php";

use Bloock\Bloock;
use Bloock\Client\AvailabilityClient;
use Bloock\Client\RecordClient;
use Bloock\Entity\Availability\IpfsLoader;

Bloock::$apiKey = getenv("API_KEY");

$recordClient = new RecordClient();
$availabilityClient = new AvailabilityClient();

$id = "publish uuid result";
$record = $recordClient->fromLoader(new IpfsLoader($id))->build();
